packages/hardware-ledger/src/LedgerKeyAgent.ts

```diff
@@ -29,17 +30,23 @@ import _LedgerConnection, {
   TransactionSigningMode,
   TxOutputDestinationType
 } from '@cardano-foundation/ledgerjs-hw-app-cardano';
-import _TransportWebHID from '@ledgerhq/hw-transport-webhid';
+import _TransportWebUSB from '@ledgerhq/hw-transport-webusb';
 import type LedgerTransport from '@ledgerhq/hw-transport';
 
-const TransportWebHID = (_TransportWebHID as any).default
-  ? ((_TransportWebHID as any).default as typeof _TransportWebHID)
-  : _TransportWebHID;
+const TransportWebUSB = (_TransportWebUSB as any).default
+  ? ((_TransportWebUSB as any).default as typeof _TransportWebUSB)
+  : _TransportWebUSB;
 const LedgerConnection = (_LedgerConnection as any).default
   ? ((_LedgerConnection as any).default as typeof _LedgerConnection)
   : _LedgerConnection;
 type LedgerConnection = _LedgerConnection;
 
+const isUsbDevice = (device: any): device is USBDevice =>
```

Comment:
The `USBDevice` class is not available in the nodejs runtime, so I decided to check a few properties.